nin/frontend/app/scripts/filters/stackFormat.js

```diff
@@ -0,0 +1,19 @@
+angular.module('nin').filter('stackFormat', function($sce) {
+  return function(stack) {
+    var lines = stack.split('\n');
+    var header = lines[0];
+    var html = `<strong>${header}</strong>\n`;
+    for(var i = 1; i < lines.length; i++) {
+      var line = lines[i];
+      line = line.trim();
+      line = line.replace(/</g, '&lt;');
+      line = line.replace(/>/g, '&gt;');
+      line = line.replace(/^at/, '<span class="muted">↳</span>');
+      line = line.replace(/https?:\/\/[^\/]+:8000/, '<em>nin</em>');
+      line = line.replace(/https?:\/\/[^\/]+:9000/, '<em>demo</em>');
+      line = line.replace(/(\(.*)/, '<span class="muted">$1</span>');
+      html += '\n' + line;
+    }
```

Comment:
I use Vivaldi, while that uses Chromium there are some differences, not sure if this specific format will be different though.